package.json

```diff
@@ -26,20 +26,20 @@
   },
   "dependencies": {
     "handlebars": "^4.5.3",
-    "i18next": "^19.0.1",
+    "i18next": "^19.0.2",
     "i18next-node-fs-backend": "^2.1.3",
     "line-reader": "^0.4.0",
     "moment": "^2.24.0",
-    "npm-check-updates": "^3.2.2"
+    "npm-check-updates": "^4.0.1"
```

Comment:
why is npu a dependency and not a dev dependency? 🤔

Reply:
Ha! The bigger question is why is it a dependency at all?

I bet at one point I forgot to do a global install for ncu....